### PR TITLE
Start work on the evaluation server.

### PR DESCRIPTION
**Pull request by Kashomon**
Right now, just read from a GCS bucket from python. A sort of hello-world.

- Write a simple binary that reads a bucket
- Add a helper to tell all the environment variables
- Fix some comments about bootstrap
- If the temp-dir for the bootstrap isn't provided, provide a default
- Set the K8S version to 1.9 (now that version aliases are in prod)
- Add a bunch of new requirements.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -77,19 +79,27 @@ def gtp(load_file: "The path to the network model files"=None,
...
+        working_dir = tempfile.mkdtemp()
+        is_temp_dir = True
+
     _ensure_dir_exists(working_dir)
     _ensure_dir_exists(os.path.dirname(model_save_path))
     dual_net.bootstrap(working_dir)
     dual_net.export_model(working_dir, model_save_path)
 
+    if is_temp_dir:
+        shutil.rmtree(working_dir)
```
Comment (amj) [inline main.py]: this gives me the heebly jeeblies.  Is it necessary often?

File: main.py [hunk 2]
```diff
@@ -77,19 +79,27 @@ def gtp(load_file: "The path to the network model files"=None,
...
+
 
 def train(
-    working_dir: 'tf.estimator working directory.',
-    chunk_dir: 'Directory where gathered training chunks are.',
-    model_save_path: 'Where to export the completed generation.',
-    generation_num: 'Which generation you are training.'=0):
+        working_dir: 'tf.estimator working directory.',
```
Comment (amj) [inline main.py]: Did the linter change these tabs all of a sudden?

File: yomi/README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Yomi Server
+
+[Yomi](https://senseis.xmp.net/?Yomi) means reading in Japanese Go Jargon, and
```
Comment (amj) [inline yomi/README.md]: s/Jargon/jargon

File: yomi/README.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+in the context of Minigo, is the name of a kubrnetes workload designed to play
+games against various engines (mostly itself).
```
Comment (amj) [inline yomi/README.md]: put in a TODO about adding a sentence or two of the architecture overview here.

File: yomi/simple_eval.py [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+import httplib2
+import json
+from oauth2client.service_account import ServiceAccountCredentials
+
+SERVICE_ACCOUNT_KEY_LOCATION = os.environ['SERVICE_ACCOUNT_KEY_LOCATION']
+BUCKET_NAME = os.environ['BUCKET_NAME']
+
+
+def run():
+    """ Talk to GCS. Do some things. """
```
Comment (amj) [inline yomi/simple_eval.py]: it does indeed look like some things are being done!

File: main.py (hunk 2 above)
Comment (Kashomon) [reply main.py]: Well, autoformat, yeah.

File: yomi/simple_eval.py (hunk 5 above)
Comment (Kashomon) [reply yomi/simple_eval.py]: very few, as of yet

File: yomi/README.md (hunk 4 above)
Comment (Kashomon) [reply yomi/README.md]: Done

File: yomi/README.md (hunk 3 above)
Comment (Kashomon) [reply yomi/README.md]: Done

File: main.py (hunk 1 above)
Comment (Kashomon) [reply main.py]: How's this?

**Pull request merged**